rust-mode-tests.el

```diff
@@ -1377,24 +1372,6 @@ this_is_not_a_string();)"
      "r\" this is a comment\n" font-lock-comment-face
      "\"this is a string\"" font-lock-string-face)))
 
-(ert-deftest font-lock-raw-string-constant ()
-  ;; There was an issue in which a multi-line raw string would be fontified
-  ;; correctly if inserted, but then incorrectly if one of the lines was then
-  ;; edited.  This test replicates how font-lock responds when text in the
-  ;; buffer is modified in order to reproduce it.
```

Comment:
This test is actually testing `rust-font-lock-extend-region`, albeit in directly.  That is, the test calls `font-lock-after-change-function`, which then calls that function.  And, this is also testing that the syntax is set correctly, but it's doing so in an indirect way, by looking to see if fontification occurred.

I could try to rewrite the test, but as-is it doesn't work, because the region extension function is gone.  Everything still works fine interactively, though.